### ISSUE TITLE
Problem viewing options

### ISSUE THREAD

**Issue by Wrda1337** [speige/WC3MapDeprotector#38]
Whatever I do, I am unable to see the "Rebuild Map" and "brute force unknown files" options, must be due to the screen ratio.
Just when I was about to test the tool for the first time! 
<img width="956" alt="problem" src="https://github.com/user-attachments/assets/4dd70ce6-ca31-400b-b064-5bfdc08a9320">

**Comment by speige** [comment]
Strange, I haven't seen this before.

Try deleting this folder, it's where extra settings are stored. They'll get re-generated next time the app is launched.
%appdata%\WC3MapDeprotector

Or maybe un-maximize and then resize based on this screenshot?
![image](https://github.com/user-attachments/assets/f9db2582-9d07-44e1-a7f7-708ed409c7ac)

**Comment by Wrda1337** [comment]
I deleted the folder and unmaximized. I have the same result.
<img width="960" alt="image" src="https://github.com/user-attachments/assets/4477446a-68c9-4041-b431-5d8917ed0fb6">
It's still an extremely useful tool, for map recovering and fixing. I've been doing it without this, though I would definetely waste less time if I start to use it.

**Comment by speige** [comment]
Do you have a really small screen resolution, like less than 1080p? That's the only other thing I can think of that would cause this.

**Comment by Wrda1337** [comment]
No I have, 1920x1080

text and icons : 150%

![image](https://github.com/user-attachments/assets/e57a1cd3-0c95-4fa6-b3e7-c77e8b27210b)

**Comment by speige** [comment]
Fala portugues? Hablo espanol

It's the 150% setting, I think. If you set it to 100% it should work. I never thought to support that setting, I'll have to see if I can fix it in a future build.

**Comment by Wrda1337** [comment]
Yes, I'm portuguese. I can sort of understand spanish, but english is definetely better, as it is my second language.
I'll check that 150%->100% setting soon, I know my programs placements will probably get ruined again but whatever, not the end of the world.

**Comment by Wrda1337** [comment]
I confirmed, it is indeed the text/program/items scaling issue! Modified to 100% and reopened the program, and I could see the icon.

**Comment by speige** [comment]
I've redone the UI in the latest version to be a lot smaller. Hopefully it'll work better for you with the 150% setting.
